data/sql/updates/pending_db_world/Fix-NPCs_Halls_of_Reflection.sql

```diff
@@ -0,0 +1,31 @@
+/*  
+SQL Dark Ranger Loralen emote
+*/  
```

Comment:
It's preferred to use -- for comments in these cases. It goes for all the comments in this file